spec/requests/api/v1/articles_spec.rb

```diff
@@ -1,32 +1,62 @@
-RSpec.describe 'GET /api/v1/articles/:id', type: :request do
+RSpec.describe Articles, type: :request do
```

Comment:
The previous format was actually better. Also, you don't have a constant named Articles. If you really need to use this naming, why not wrap it in quotes?

Reply:
But okay, I see what you are doing below... I would call this "Articles endpoint" or something like that.

Reply:
Request specs for ArticlesController split into `index_specs.rb` and `show_specs.rb`.